mappings/net/minecraft/test/StartupParameter.mapping

```diff
@@ -1,4 +1,239 @@
+		ARG 1 pos
+		ARG 2 property
+		ARG 3 predicate
+		ARG 4 errorMessage
+	METHOD method_35989 (Lnet/minecraft/class_2338;Lnet/minecraft/class_3341;Lnet/minecraft/class_2338;)V
+		ARG 3 checkedPos
+	METHOD method_35990 expectBlockWithStateOf (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)V
+		ARG 1 checkedPos
+		ARG 2 correctStatePos
+	METHOD method_35991 checkBlock (Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Ljava/lang/String;)V
+		ARG 1 pos
+		ARG 2 predicate
+		ARG 3 errorMessage
+	METHOD method_35992 checkBlock (Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Ljava/util/function/Supplier;)V
+		ARG 1 pos
+		ARG 2 predicate
+		ARG 3 errorMessageSupplier
+	METHOD method_35993 addFinalTask (Ljava/lang/Runnable;)V
+		ARG 1 runnable
+	METHOD method_35999 (Ljava/util/function/Predicate;Lnet/minecraft/class_2680;)Z
+		ARG 1 state
+	METHOD method_36001 killAllEntities ()V
+	METHOD method_36002 toggleLever (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_36003 waitAndRun (JLjava/lang/Runnable;)V
+		ARG 1 ticks
+		ARG 3 runnable
+	METHOD method_36004 dontExpectEntity (Lnet/minecraft/class_1299;)V
+		ARG 1 type
+	METHOD method_36005 dontExpectEntityToTouch (Lnet/minecraft/class_1299;DDD)V
+		ARG 1 type
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+	METHOD method_36006 spawnMob (Lnet/minecraft/class_1299;FFF)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36007 spawnMob (Lnet/minecraft/class_1299;III)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36008 spawnMob (Lnet/minecraft/class_1299;Lnet/minecraft/class_243;)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 vec3d
+	METHOD method_36009 spawnMob (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_36011 dontExpectBlock (Lnet/minecraft/class_2248;III)V
+		ARG 1 block
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36012 dontExpectBlock (Lnet/minecraft/class_2248;Lnet/minecraft/class_2338;)V
+		ARG 1 block
+		ARG 2 pos
+	METHOD method_36013 (Lnet/minecraft/class_243;Lnet/minecraft/class_1297;)Z
+		ARG 1 entity
+	METHOD method_36014 getBlockEntity (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2586;
+		ARG 1 pos
+	METHOD method_36017 checkBlockState (Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Ljava/util/function/Supplier;)V
+		ARG 1 pos
+		ARG 2 predicate
+		ARG 3 errorMessageSupplier
+	METHOD method_36018 addInstantFinalTask (Ljava/lang/Runnable;)V
+		ARG 1 runnable
+	METHOD method_36021 spawnPlayer ()Lnet/minecraft/class_1657;
+	METHOD method_36022 expectEntityAt (Lnet/minecraft/class_1299;III)V
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36023 expectEntityAt (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)V
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_36024 expectBlockAtTheEnd (Lnet/minecraft/class_2248;III)V
```

Comment:
```suggestion
	METHOD method_36024 expectBlockAtEnd (Lnet/minecraft/class_2248;III)V
```
`TheEnd` conflicts with the end dimension and an article `the` isn't necessary for naming.